### PR TITLE
DA-4639 & DA-4644: Add GITHUB_CATEGORY env var in the dockerfile

### PR DESCRIPTION
**Pull request by linonymous**
- add GITHUB_CATEGORY for issues & pull requests connectors

Signed-off-by: linonymous <iamlinonymous@gmail.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,6 +2,7 @@ FROM alpine:3.14
 WORKDIR /app
 ENV GITHUB_ORG='<GITHUB-ORG>'
 ENV GITHUB_REPO='<GITHUB-REPO>'
+ENV GITHUB_CATEGORY='<GITHUB-CATEGORY>'
```
Comment (lukaszgryglicki) [inline Dockerfile:5]: maybe use "CATEGORIES" instead of "CATEGORY"?

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm - one suggestion to match cmd line parameter name with env var var name.

File: Dockerfile (hunk 1 above)
Comment (linonymous) [reply Dockerfile]: done.

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm

**Pull request merged**